static/bidder-info/bluesea.yaml

```diff
@@ -3,9 +3,15 @@ maintainer:
   email: prebid@blueseasx.com
 endpointCompression: gzip
 modifyingVastXmlAllowed: true
+gvlVendorID: 1294
```

Comment:
Verified:
"1294": {
      "id": 1294,
      "name": "Bluesea Holdings Hong Kong Limited",
      "purposes": [1],
      "legIntPurposes": [],
      "flexiblePurposes": [],
      "specialPurposes": [],
      "features": [],
      "specialFeatures": [],
      "cookieMaxAgeSeconds": null,
      "usesCookies": false,
      "cookieRefresh": false,
      "usesNonCookieAccess": true,
      "dataRetention": {
        "purposes": {

        },
        "specialPurposes": {

        }
      },
      "urls": [
        {
          "langId": "en",
          "privacy": "https://www.blueseasx.com/policy.html",
          "legIntClaim": "https://www.blueseasx.com/policy.html"
        }
      ],
      "dataDeclaration": [1, 2, 3, 4, 8, 11],
      "deviceStorageDisclosureUrl": "https://www.blueseasx.com/tcf-disclosure.json"
    },

Reply:
Sorry for the delay. LGTM. Merged.